app/subcommands/repo/list

```diff
@@ -39,7 +39,9 @@ for dir in "$DAB_CONF_PATH"/repo/*; do
 		fi
 	fi
 
-	repo_row "$repo" "$status" "${clean:-}" "${uptodate:-}"
+	url="$(dab config get repo/"$repo"/website)"
+
+	repo_row "$repo" "$status" "${clean:-}" "${uptodate:-}" "$url"
```

Comment:
will this cause an error if the `website` key for a given repo is not defined? I think the behaviour desired in this case is for the column to be empty on rows of repos that do not define it.

Reply:
I believe that if the key is not defined the `dab config get` will return an empty string.

Reply:
Actually looking at the source for the `config get` command, the command does not return (print) anything if the key is not defined.